springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po

```diff
@@ -26,7 +25,7 @@ msgstr ""
 
 #: springboard_iogt/utils.py:20
 msgid "Become a U-Reporter & start sharing!"
-msgstr ""
+msgstr "Voice matters"
```

Comment:
@TrevorOctober this doesn't look like the right Arabic translation for `Become a U-Reporter & start sharing!`